Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM php:7.3-apache
+
+RUN apt-get update -y
+RUN apt-get install -y sendmail libpng-dev zlib1g-dev libc-client-dev libkrb5-dev libicu-dev libzip-dev zip
+
+RUN docker-php-ext-install mysqli
```

Comment:
Same for all of these.

Reply:
Thank you for the tips. I'm already sending a new pull request.